src/Kunstmaan/NodeBundle/Resources/views/NodeAdmin/Modals/_pagetemplate.html.twig

```diff
@@ -0,0 +1,34 @@
+{% import _self as macros %}
+
+{% for row in pageTemplate.rows %}
+<div class="row">
```

Comment:
But is it a difference? I mean, browser will still render the same regarding to indentation of generated source received from the server.